pages/login.js

```diff
@@ -24,6 +33,29 @@ export default function Login() {
 
     console.log("Username: %s", username)
     console.log("Password: %s", password)
+
+    // Send username and password to API for validation
+    let user_credentials = {
+      "username" : username,
+      "password" : password
+    }
```

Comment:
Nitpick - you can just put the variable into the object. JS will automatically make the key = the variable name and the value = variable value.

```suggestion
    let user_credentials = {
      username,
      password
    }
```